docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -5,36 +5,42 @@ description: Learn how to deploy your Prisma database service to Digital Ocean m
+<InfoBox>
+
+The setup described in this section does not include features you would normally expect from production-ready servers, such as _automated backups_ and _active failover_. We will add more guides in the future describing a more complete setup suitable for production use.
 
 </InfoBox>
 
-This section describes how to set up a fully functioning Prisma server on Digital Ocean in less than 20 minutes.
+## 1. Register at Digital Ocean
 
-In the following procedure you will connect to your droplet with ssh, and manually install all required dependencies. Most users should use the procedure described in [Digital Ocean (Docker Machine)](!alias-texoo9aemu) instead.
+<Instruction>
+
+If you haven't already, you need to start by creating an account at for Digital Ocean. You can do so [here](https://cloud.digitalocean.com/registrations/new).
 
-Digital Ocean is an easy to use provider of virtual servers. They offer different compute sizes called droplets. The 10$ droplet is a good starting point for a Prisma server.
+</Instruction>
 
-> The setup described in this section does not include features you would normally expect from production ready servers, such as backup and active failover. We will add more guides in the future describing a more complete setup suitable for production use.
 
-## Creating a droplet
+## 2. Create your Digital Ocean Droplet
 
-If you are new to Digital Ocean you should start by creating an account at https://www.digitalocean.com/
+You can now go ahead and create your Droplet. First pick the latest Ubuntu Distribution, as of this writing it is `18.04.1 x64`. Next pick a region that is close to your end user. If this Prisma will be used for development, you should pick the region closest to you. Set a name for your droplet, `prisma` for example.
 
-After you log in, create a new Droplet:
+Relax while your droplet is being created. This should take less than a minute.
 
-1. Pick the latest Ubuntu Distribution. As of this writing it is 17.10.
-2. Pick a standard droplet. The 10$/mo droplet with 1 GB memory is a good starting point.
-3. Pick a region that is close to your end user. If this Prisma will be used for development, you should pick the region closest to you.
-4. Create an ssh key that you will use to connect to your droplet. Follow the guide provided by Digital Ocean.
-5. Set a name for your droplet. For example `prisma`
-6. Relax while your droplet is being created. This should take less than a minute.
+Once you have your droplet available, you need to generate a ssh key which is needed to connect to your Droplet
 
-## Installing Prisma
+<Instruction>
 
-### Connect
+Follow the instructions [here](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/) to generate your ssh key
+
+</Instruction>
+
+## 3. Install Prisma on your Droplet
 
 Digital Ocean will show you the ip address. If you are using MacOS or Linux you can now connect like this:
```

Comment:
This is very vague. How/where does it show the IP address? Maybe provide a screenshot?